src/stories/Table/Table.stories.tsx

```diff
@@ -93,6 +97,6 @@ Table.args = {
     options: {
         disableSortRemove: true,
         autoResetSortBy: true,
-        disableMultiSort: true
+        disableMultiSort: false
```

Comment:
Let's add two more stories here to show:
1.) when multisort is enabled
2.) when custom widths are passed.